pkgs/top-level/all-packages.nix

```diff
@@ -9567,6 +9567,8 @@ julia_15 = callPackage ../development/compilers/julia/1.5.nix {
 
   kotlin = callPackage ../development/compilers/kotlin { };
 
+  kotlin-language-server = callPackage ../development/tools/kotlin-language-server { };
```

Comment:
This does not belong into compilers. Please put it under software development tools or similar.

Reply:
Um, this is the ‘kotlin-language-server’ package, not the ‘kotlin’ package.